api/controllers/decisions/cohDecisions.ts

```diff
@@ -1,6 +1,6 @@
 import * as express from 'express'
-
-const { getHearingIdOrCreateHearing, getDecision, postDecision, putDecision } = require('../../services/coh-cor-api/coh-cor-api')
+const {getHearingIdOrCreateHearing, getDecision, postDecision, putDecision} = require('../../services/coh-cor-api/coh-cor-api')
+const {relistHearing} = require('../../services/coh')
```

Comment:
I thought we should not be using require but import?

Reply:
@ficklephil can you mark this as solve